cfgov/ask_cfpb/templates/admin/export.html

```diff
@@ -0,0 +1,22 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% block titletag %}{% trans "Export Ask data" %}{% endblock %}
+
+{% block content %}
+
+    {% include "wagtailadmin/shared/header.html" with title="Ask CFPB Export" icon="download" %}
+
+    <div class="nice-padding">
+        <h3>
+            Download a spreadsheet of all Ask CFPB answers. <br/>
+            This may take up to a minute or so to complete.
+        </h3>
+        <form method="post">
+            {% csrf_token %}
+            <button type="submit" name="Export CSV" class="button"/>
```

Comment:
`<button` has erroneous `/>` at end. Should be `>`.

Reply:
Also, I'd avoid the `name` attribute having a space in it. Is this format needed on the backend? Could it be `name="export-csv"` or similar?